src/scenes/Partners/Detail/PartnerDetail.tsx

```diff
@@ -87,23 +87,24 @@ export const PartnerDetail = () => {
 
 const PartnerHeader = ({ partner, editPartner }: PartnerViewEditProps) => {
   const partnerName = partner?.organization.value?.name.value;
-  const abrev =
-    partnerName && partnerName.length >= 30
-      ? partner.organization.value.avatarLetters
-      : null;
```

Comment:
Just want to double check that we are intentionally going away from the "long name is shown smaller with a large abbreviation"?

Reply:
I'm not sure.  I thought this code that uses the avatar letters in place of the acronym was just a place holder until the acronym was implemented.